packages/commonwealth/server/controllers/server_communities_methods/delete_community.ts

```diff
@@ -122,9 +117,7 @@ export async function __deleteCommunity(
             },
           );
 
-          const models: Array<
-            ModelStatic<ModelInstance<{ community_id?: string }>>
-          > = [
+          const models: ModelStatic<Model>[] = [
```

Comment:
This isn't the correct type -> the type for this should be any model that has `community_id` defined which this type does not enforce.

Reply:
fixed